hpat/hiframes/hiframes_typed.py

```diff
@@ -907,19 +907,19 @@ def run_call_series_quantile_default(A):
                     data, index, name),
                 [data, index, name], pre_nodes=nodes)
 
-        if func_name == 'pct_change':
-            nodes = []
-            data = self._get_series_data(series_var, nodes)
-            # TODO: support default period argument
-            if len(rhs.args) == 0:
-                args = [data]
-                func = series_replace_funcs[func_name + "_default"]
-            else:
-                assert len(rhs.args) == 1, "invalid args for " + func_name
-                shift_const = rhs.args[0]
-                args = [data, shift_const]
-                func = series_replace_funcs[func_name]
-            return self._replace_func(func, args, pre_nodes=nodes)
+        # if func_name == 'pct_change':
```

Comment:
You are likely to break the multiprocessing parallelism by disabling "old style" implementation.